pyproject.toml

```diff
@@ -42,6 +40,14 @@ http2 = [
 socks = [
     "socksio==1.*",
 ]
+trio = [
+    "trio>=0.21.0,<0.22.0",
```

Comment:
Is this necessary?

```suggestion
    "trio>=0.21.0",
```

Reply:
This depends on the AnyIO version used. AnyIO 4 will only work with Trio 0.22 or later while AnyIO 3 will only work with 0.21 or earlier. AnyIO itself has the `trio` extra which will constraint the version appropriately.